src/App.js

```diff
@@ -1,33 +1,51 @@
 import {useMemo, useState} from 'react';
 import styled from 'styled-components';
 
-import Categories from './components/categories';
+import ColorCategoryFilter from './components/colorCategoryFilter';
 import ImageContainer from './components/imageContainer';
 import data from './garmentsData';
 
 export default function App() {
   const [garments] = useState(data);
-
   const [colorList, setColorList] = useState();
+  const [brandList, setBrandList] = useState();
+  const [favorite, setFavorite] = useState([]);
 
   function handleColorChange(event) {
     setColorList(event.target.value);
   }
+  function handleBrandChange(event) {
+    setBrandList(event.target.value);
+  }
 
-  function getFilteredList() {
+  function getFilteredColorList() {
     if (!colorList) {
       return garments;
     }
     return garments.filter(item => item.color === colorList);
   }
 
-  let filteredColorList = useMemo(getFilteredList, [colorList, garments]);
+ function getFilteredBrandList() {
+    if (!brandList) {
+      return garments;
+    }
+    return garments.filter(item => item.brand === brandList);
+  }
+
+  let filteredColorList = useMemo(getFilteredColorList, [colorList, garments]);
+ // let filteredBrandList = useMemo(getFilteredBrandList, [brandList, garments]);
+
+  const addToFavorites = id => {
```

Comment:
Does this function work, did you try it 🤔
There are a few ways you can **add** and **remove** your favorites - I recommend you take another look at your notes from the respective sessions or ask your classmates how the implement the logic.